client/src/App.js

```diff
@@ -8,13 +8,16 @@ import {
 } from "react-router-dom";
 import HomePage from "./Components/HomePage/HomePage";
 import NoPageFound from "./Components/404";
+import UploadPost from "./Components/uploadPost.js/uploadPost";
```

Comment:
use first letter capital for folder and file names

Reply:
Ok Yash i changed the naming and i am yet to raise a PR